apps/site/src/app/schedule/ClipboardSchedule/ClipboardSchedule.tsx

```diff
@@ -56,7 +58,18 @@ interface ClipboardScheduleProps {
 	}[][];
 }
 
+// 10/4/23 10AM in UTC
+const hackingStarts = new Date(Date.UTC(2023, 10, 4, 17, 0, 0));
+const hackingEnds = new Date(Date.UTC(2023, 10, 5, 5, 0, 0));
```

Comment:
Issue: end time doesn't seem right. While DST will end, shouldn't affect us since hacking ends is before 2am.